LICENSE

```diff
@@ -0,0 +1,201 @@
+ Apache License
```

Comment:
I think this needs to be indented/centered (I would copy/paste from a license file from a different app so it matches spacing exactly)